test/PerformanceTests/SerializationComparisonsTests/Lib/DataModel.cs

```diff
@@ -14,6 +19,11 @@ public class Customer
         public int Id { get; set; }
         public string Name { get; set; }
         public List<string> Emails { get; set; }
+        public string Bio { get; set; }
+
+        [JsonConverter(typeof(Base64Converter))]

```

Comment:
Curious why we need a JsonConverter here

Reply:
`JsonSerializer` does not seem to serialize `byte[]` fields by default.